assume/scenario/loader_csv.py

```diff
@@ -563,14 +564,14 @@ def setup_world(
     config = scenario_data["config"]
     start = scenario_data["start"]
     end = scenario_data["end"]
-    index = scenario_data["index"]
     powerplant_units = scenario_data["powerplant_units"]
     storage_units = scenario_data["storage_units"]
     demand_units = scenario_data["demand_units"]
     dsm_units = scenario_data["dsm_units"]
     forecaster = scenario_data["forecaster"]
 
-    save_frequency_hours = config.get("save_frequency_hours", 48)
+    # save every thousand steps by default to free up memory
+    save_frequency_hours = config.get("save_frequency_hours", 1000)
```

Comment:
Did we check the writing of dfs in the output role with a high save_frequency?

Reply:
@maurerle thanks for the analysis! But 350mb compared to 500 mb, so 150mb extra is a low price to price for a significant speed up. In my opinion this change to longer periods is justified. I don't think any more or less modern system would run out of memory because of this.

Reply:
On my machine it does not give a significant speed up by reducing this param. How much is it on yours?
Typically, the save_dfs function should not block anyway.

The problem is, that this increases linearly, I just stopped the simulation after some time.

Reply:
@maurerle try activating the saving to csv files, this will change the timing for you

Reply:
@maurerle also as of now on pandas_wrapper the config for example_03 has the save frequency as null, so it is never saved

Reply:
@nick-harder CSV does only save at the end as `save_frequency_hours is disabled due to CSV export being enabled. Data will be stored in the CSV files at the end of the simulation.`

Which gives above graphs as well when setting `save_frequency_hours: 96` in the yaml.
So I wonder how you see a speedup with this setting

Reply:
Which we did as `unit_dispatch` changes in the CSV files when the accepted_* fields are added for example.
so eventually we should fix this as I do not really like to have linearly increasing memory consumption for a simulation tool :)

Saving csv is indeed a little slower, when the above mentioned handler to switch back to None is removed.